_posts/2019/2019-05-21-making-a-move-abroad-from-bolivia-to-hamburg.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: Making a move abroad - from Bolivia to Hamburg
+date: 2019-05-21
+header_image: public/relocation.jpg
+header_position: center
+header_overlay: true
+category: on-the-job
+tags: ["onboarding", "relocation", "new job"]
+authors: ["Kathia"]
+about_authors: ["ksalazar"]
+---
+
+Hi, my name is Kathia. 
+I come from Bolivia, and since March 2019 I'm working as a Software Developer at ePages in Hamburg.
+In late 2018 I applied online, and then had a first interview with Serena, the HR recruiter.
+After the interview, I had to solve a coding challenge.
+The next week, I talked with Unai, one of the developers of my current team, to discuss a few things about my code as well as my previous experience.
+This was followed by an interview with Harm, the CTO, and Serena.
+Finally, I received the offer from ePages 10 days before Christmas.
+So, this was like an early Christmas present for me 🙂.
+
+## Why the decision was easy
+
+I learnt German in Bolivia, and reached a pretty decent level - B1 😉. 
```

Comment:
How comes you learnt German in Bolivia? Just out of interest? Some background would be nice to read.